phpunit.xml.dist

```diff
@@ -8,6 +8,9 @@
     <filter>
         <whitelist processUncoveredFilesFromWhitelist="true">
             <directory suffix=".php">./src</directory>
+            <exclude>
+              <file>./src/SourceLocator/SourceStubber/PhpStormStubsMap.php</file>
```

Comment:
Is it worth excluding this at all?

Reply:
Removed. It's not reported as "no coverage" anymore.